packages/syft/src/syft/core/adp/vm_private_scalar_manager.py

```diff
@@ -34,16 +38,21 @@ class VirtualMachinePrivateScalarManager(RecursiveSerde):
 
     def __init__(self) -> None:
         self.prime_factory = PrimeFactory()
-        self.prime2symbol = {}
+        self.prime2symbol: Dict[Any, Any] = {}
 
     def get_symbol(
         self,
-        min_val: AcceptableSimpleType,
-        value: AcceptableSimpleType,
-        max_val: AcceptableSimpleType,
+        min_val: Union[bool, int, float],
```

Comment:
Are these types broader like np.int np.float, should we introduce a more generic numeric type?